src/log4stash/ElasticSearchAppender.cs

```diff
@@ -85,19 +87,35 @@ public override void ActivateOptions()
 
             if (Template != null && Template.IsValid)
             {
-                _client.PutTemplateRaw(Template.Name, File.ReadAllText(Template.FileName));
+                if (!Template.ForcePut)

```

Comment:
Just remove the `if` and leave ` _finishedPuttingTemplate = !Template.ForcePut;`